tests/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:13.0.1-alpine AS buildstep
+LABEL maintainer "RPS <devops@rpsgroup.com>"
+
+RUN mkdir -p /web
+WORKDIR /web
+
+COPY ./web/ /web/
+RUN yarn global add grunt-cli && \
+    yarn install && \
+    grunt
```

Comment:
Not sure we need the node build for the python tests. See if we can drop this step to make it a bit faster to run

Reply:
Skipping that step gets us a wee bit faster.